### PR TITLE
feat: add path safe KPI and subdim names for CSV download

### PR DESCRIPTION
**Pull request by Samyak2**

### CODE REVIEW

File: chaos_genius/views/kpi_view.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
 from chaos_genius.settings import SUMMARY_DEEPDRILLS_ENABLED_TIME_RANGES
 from chaos_genius.utils.pagination import pagination_args, pagination_info
 from chaos_genius.utils.search import SEARCH_PARAM_NAME, make_search_filter
+from chaos_genius.utils.utils import make_path_safe
```
Comment (manassolanki) [inline chaos_genius/views/kpi_view.py:44]: Do we need this?

File: chaos_genius/views/anomaly_data_view.py [hunk 2]
```diff
@@ -431,8 +436,13 @@ def _get_dimensions_values(
         {
             "label": dimension,
             "value": dimension,
+            "label_path_safe": make_path_safe(dimension),
             "subdim_value_options": [
-                {"label": value, "value": value}
+                {
+                    "label": value,
+                    "value": value,
+                    "label_path_safe": make_path_safe(value),
```
Comment (manassolanki) [inline chaos_genius/views/anomaly_data_view.py:444]: Is this being used at any other place?

**Review by manassolanki — CHANGES_REQUESTED** [review]

File: chaos_genius/views/kpi_view.py (hunk 1 above)
Comment (Samyak2) [reply chaos_genius/views/kpi_view.py:44]: No, it's not needed. Got left over from another change.

File: chaos_genius/views/anomaly_data_view.py (hunk 2 above)
Comment (Samyak2) [reply chaos_genius/views/anomaly_data_view.py:444]: It is used by the frontend to make the CSV name and nowhere else.

**Pull request merged**